Feat: health check docker (#4467)
jina-ai/jina: docs/how-to/docker-compose.md

All Executors in the Flow should be used with `jinahub+docker://...` or `docker://...`.
```

```{caution}

> If you are using Executor which rely on docker image built with a jina version anterior at 3.1.3, please remove the 

Reviewer: ```suggestion
If you are using Executor which rely on docker image built with a jina version prior to 3.1.3, please remove the 
```
